Cargo.toml

```diff
@@ -100,7 +100,7 @@ rustls-pemfile = { version = "2", default-features = false }
 tokio-rustls = { version = "0.26.0", default-features = false }
 anyhow = { version = "1", default-features = false }
 chrono = { version = "0.4", default-features = false, features = ["clock"] }
-ubyte = { version = "0.10", default-features = false, features = ["serde"] }
+bytesize = { version = "1.3.0", default-features = false, features = ["serde"] }
```

Comment:
Switched to `bytesize` because `ubyte` has weird behavior with how it decides to format byte values in terms of switching between SI and non-SI prefixes, which means sometimes you get kB and sometimes you get KiB, and you have to be looking closely to understand why one number might bigger or smaller than another.

`ubyte` has no way to easily force using SI prefixes, but `bytesize` does.